components/micro-gateway-core/src/main/ballerina/src/gateway/utils/throttle_util.bal

```diff
@@ -172,3 +172,16 @@ public function getGlobalThrottleStream() returns stream<GlobalThrottleStreamDTO
 public function getIsStreamsInitialized() returns boolean {
     return isStreamsInitialized;
 }
+
+public function getResourceTier(string resourceName) returns string {
+    TierConfiguration? tier = resourceTierAnnotationMap[resourceName];
+    string? policy = (tier is TierConfiguration) ? tier.policy : ();
+    if (policy is string) {
+        if (policy.length() == 0) {
+            return UNLIMITED_TIER;
+        } else {
```

Comment:
FYI, We can just `return policy` here without the else block

Reply:
Fixed